modules/misc/numlock.nix

```
@@ -23,6 +23,7 @@ in
 
       Service = {
         Type = "oneshot";
+        RemainAfterExit = "yes";
```

Can use
```suggestion
        RemainAfterExit = true;
```
for a slightly cleaner code. Same with xsession below.

---

Done